permissions/plugin-azure-publishersettings-credentials.yml

```diff
@@ -8,6 +8,7 @@ paths:
 developers:
 - "batmat"
 - "markwm"
+- "company-cloudbees-developers"
```

Comment:
this plugin is dead ftr, it was emptied out: https://github.com/jenkinsci/azure-publishersettings-credentials-plugin